docs/advanced-entity-analytics/turn-on-risk-engine.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[turn-on-risk-engine]]
+== Turn on the risk scoring engine
+
+beta[]
+
+IMPORTANT: To use Entity Risk Scoring, your role must have the appropriate privileges. For more information, refer to 
+
+// <<ers-requirements, Entity Risk Scoring prerequisites>>.
+
+The latest risk scoring engine, available in {stack} version 8.11 and newer, runs hourly to aggregate `Open` and `Acknowledged` <<alerts-ui-manage, alerts>> from the last 30 days, and assigns risk score to the host or user. It then aggregates the individual risk scores and normalizes them to a 0-100 range. The engine assigns a risk level by mapping the normalized risk score to one of these levels:
```

Comment:
We don't typically identify the version in which each feature is introduced. Also, all the emphasis on this being "the latest" feels like we're trying to convince users to switch to the latest & greatest, which isn't our typical tone in docs. I also feel like we're missing some overall explanation of the two engines: is the new one replacing the old one, or are they both available? This is kind of explained later in the topic, but if we're going to keep calling this "the latest" we should probably establish earlier on what that means.

This info also feels out of place in the topic's opening paragraph: the page is supposed to tell us how to turn on the engine, but this is more like "how it works."